{{cookiecutter.repostory_name}}/noxfile.py

```diff
@@ -20,25 +19,37 @@
     nox.options.force_venv_backend = 'none'
 
 
+def run_readable(session, mode="fmt"):
+    session.run(
+        'docker',
+        'run',
+        '--rm',
+        '-v', f'{ROOT.absolute()}:/data',
+        '-w', '/data',
+        '-u', f'{os.geteuid()}:{os.getegid()}',
+        'ghcr.io/bobheadxi/readable:v0.5.0@sha256:423c133e7e9ca0ac20b0ab298bd5dbfa3df09b515b34cbfbbe8944310cc8d9c9',
```

Comment:
![deja-vu](https://github.com/reef-technologies/cookiecutter-rt-django/assets/73276794/0f81c8fb-8371-4063-b0be-04bd4e4c2626)

Reply:
I don't believe there is anything we can do about this, but I'm open to suggestions